src/main/java/com/google/alpollo/SongDataBase.java

```diff
@@ -2,15 +2,47 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** Class that provides methods to work with the database. */
 public class SongDataBase {
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final String ARTIST_PROPERTY = "artist";
+  private static final String NAME_PROPERTY = "name";
+  private static final String ALBUM_PROPERTY = "album";
+  private static final String COUNTER_PROPERTY = "counter";
```

Comment:
It is the number of times song was requested. I can add it to documentation because I don't know how to call it in another way.

Reply:
Maybe `SEARCH_COUNTER`? I think in any case it deserves a documentation, because it will be better to explain what it does. "Number of times users searched for this song in the system.", or something similar.